units/recalculate_change_points_test.go

```diff
@@ -129,15 +134,47 @@ func makePerfResultsWithChangePoints(unique string, seed int64) ([]testResultsAn
 	}
 
 	// time to map over change points
-	changePoints := map[string][]int{}
+	changePoints := map[string][]changePointIndexPercentChange{}
 	for measurement, cpIndices := range timeSeriesChangePoints {
 		measurementName := "measurement_" + strconv.Itoa(measurement)
-		changePoints[measurementName] = append(changePoints[measurementName], cpIndices...)
+		cpsWithIndexAndPercentChange := make([]changePointIndexPercentChange, len(cpIndices))
+		currentTimeSeries := timeSeries[measurement]
+		for i, pointIndex := range cpIndices {
```

Comment:
this feels like we're duplicating production logic in a test.

Reply:
This was changed